app/src/main/java/org/fossasia/pslab/adapters/ControlMainAdapter.java

```diff
@@ -13,17 +17,37 @@
  * Created by asitava on 6/6/17.
  */
 
-public class ControlMainAdapter extends RecyclerView.Adapter<ControlMainAdapter.ViewHolder> {
+public class ControlMainAdapter extends RecyclerView.Adapter<ControlMainAdapter.ViewHolder> implements View.OnClickListener{
```

Comment:
removing it won't make a difference because you are not doing anything in ```onClick``` Line 49. So try removing this ```implements View.OnClickListener``` shouldn't make a difference.